### PR TITLE
MuhammadFareezIqmal.IIUMSchedule version 0.1.0.8

### PR DESCRIPTION
**Pull request by iqfareez**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)? (1.2)

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69843)

### CODE REVIEW

**Comment by iqfareez** [comment]
Blocking https://github.com/microsoft/winget-pkgs/issues/68845#issuecomment-1216812029

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 0.1.0.8
```
Comment (ItzLevvie) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.installer.yaml:5]:
```suggestion
PackageVersion: 0.1.0.7
```
This must match Control Panel.

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 0.1.0.8
```
Comment (ItzLevvie) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.1.0.7
```
This must match Control Panel.

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 0.1.0.8
```
Comment (ItzLevvie) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.yaml:5]:
```suggestion
PackageVersion: 0.1.0.7
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.yaml (hunk 3 above)
Comment (iqfareez) [reply manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.yaml:5]: Ahh. How can I miss this one. I'll reupload a new installer with updated version. Is that okay?

Comment (ItzLevvie) [reply manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.yaml:5]: Sure.

**Review by ItzLevvie — DISMISSED** [review]

**Comment by iqfareez** [comment]
Thanks

**Comment by iqfareez** [comment]
Any suggestion on how to fix the `Validation-Executable-Error` error?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**